packages/core/src/inspection.ts

```diff
@@ -51,7 +50,7 @@ export interface InspectedEventEvent extends BaseInspectionEventProperties {
   // The source might not exist, e.g. when:
   // - root init events
   // - events sent from external (non-actor) sources
-  sourceRef: AnyActorRef | undefined;
+  sourceRef: ActorRefLike | undefined;
```

Comment:
u might want to add some changeset for this change as without it u wont release `xstate` package and `@xstore/store` won't be able to utilize this improvement

Reply:
[a32f1e8](https://github.com/statelyai/xstate/pull/5079/commits/a32f1e818bfdaef356bbdbf1720ac61b31659b2c)